Cython/Utility/CppConvert.pyx

```diff
@@ -52,14 +52,24 @@ cdef vector[X] {{cname}}(object o) except *:
 #################### vector.to_py ####################
 
 cdef extern from *:
-    cdef cppclass vector "const std::vector" [T]:
+    cdef cppclass vector "std::vector" [T]:
```

Comment:
In all other cases const is added in the function signature instead. Is there a reason, why it has to be in here for std::vector?

Reply:
Probably just predates proper `const` support.